scripts/busca.js

```diff
@@ -76,17 +76,24 @@ module.exports = function(robot) {
         res.reply(':thinking_face: I can\'t find any upcoming events for ' + name + '.');
         return;
       }
-
+      edates = [];
       response = ':spiral_calendar_pad: Looks like ' + name
         + ' has the following ' + numEvents + ' events:';
-
       eventKeys.forEach(function(key) {
         var event = data[key],
             summary = event.summary,
             startDate = event.start.toDateString(),
             endDate = event.end.toDateString();
-        response += '\n- ' + summary + ' from ' + startDate + ' - ' + endDate;
+            edates.push(event);
+
+        //response += '\n- ' + summary + ' from ' + startDate + ' until ' + endDate;
```

Comment:
We can go ahead and remove this comment. It doesn't clarify any code and we can grab the history directly from git.